airflow-core/src/airflow/ui/public/i18n/locales/pt/common.json

```
@@ -0,0 +1,280 @@
+{
+  "admin": {
+    "Config": "Configuração",
+    "Connections": "Conexões",
+    "Plugins": "Plugins",
+    "Pools": "Pools",
+    "Providers": "Fornecedores",
```

```suggestion
    "Providers": "Providers",
```
I think I would keep "Providers" here.

---

yeah, kept